netbeans/build.gradle

```diff
@@ -17,7 +17,7 @@ String netbeansVersion = 'RELEASE80'
 repositories {
     mavenLocal()
     mavenCentral()
-    maven { url 'http://nexus.gluonhq.com/nexus/content/repositories/releases/' }
+    maven { url 'https://nexus.gluonhq.com/nexus/content/repositories/releases/' }
     maven { url 'http://bits.netbeans.org/nexus/content/groups/netbeans' }
     maven { url 'http://bits.netbeans.org/maven2/' }
     maven { url 'http://repo.gradle.org/gradle/libs-releases-local' }
```

Comment:
We can change the other maven urls to https as well? Only bits.netbeans.org doesn't seem to serve https.